src/Prepr.php

```diff
@@ -260,19 +264,25 @@ public function autoPaging()
     {
         $this->method = 'get';
 
-        $perPage = 10;
+        $perPage = 100;
```

Comment:
10 is erg weinig voor bijvoorbeeld 1300 publicaties. Dit komt neer op 130*30ms = 3.9seconden aan network latency en 130 api calls.
Ik denk dat we deze even moeten afstemmen met Tim.

Reply:
Klopt, dit had ik gedaan voor te testen